blinkit-app/src/homePage/atoms/insertSelect/InsertSelect.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import { UserConsumer } from "../../../userContext";
+
+const addOption = (value, index) => {
+  if (index === 0)
+    return (
+      <option key={"option-" + index} value={value} hidden>
+        {value}
+      </option>
+    );
+  return (
+    <option key={"option-" + index} value={value}>
+      {value}
+    </option>
+  );
+};
+
+export class InsertSelect extends Component {
+  render() {
+    const values = this.props.values;
+
+    return (
+      <UserConsumer>
```

Comment:
Why do we need context here?
If we need certain values here, the parent component will pass it. The use-case of this wrapper is to take in options and create a select input

Reply:
This is meant to be a generic select input